src/browser/auto-updater-linux.coffee

```diff
@@ -0,0 +1,115 @@
+{EventEmitter} = require 'events'
+_ = require 'underscore-plus'
+ChildProcess = require 'child_process'
+app = require 'app'
+
+distros = {
+  ubuntu: /(\S*ubuntu)/i,
+  debian: /(debian)/i,
+  redhat: /(red(\s?)hat)/i,
+  centos: /(centos)/i,
+  fedora: /(fedora)/i,
+}
+
+spawn = (command, args, callback) ->
+  stdout = ''
+
+  try
+    spawnedProcess = ChildProcess.spawn(command, args)
+  catch error
+    # Spawn can throw an error
+    process.nextTick -> callback?(error, stdout)
+    return
+
+  spawnedProcess.stdout.on 'data', (data) ->
+    stdout += data
+
+  error = null
+  spawnedProcess.on 'error', (processError) -> error ?= processError
+  spawnedProcess.on 'close', (code, signal) ->
+    error ?= new Error("Command failed: #{signal ? code}") if code isnt 0
+    error?.code ?= code
+    error?.stdout ?= stdout
+    callback?(error, stdout)
+
+class AutoUpdater
+  _.extend @prototype, EventEmitter.prototype
+
+  stdout = ChildProcess.execSync('cat /etc/issue').toString().replace(/\\./g, "")
```

Comment:
I believe Atom uses `lsb_release` to determine the OS - could you do that instead?
https://github.com/atom/notifications/blob/4716ca24f74c685f6db85447e432e8a173d86a67/lib/user-utilities.coffee#L64-L78 (though it does seem a bit broken on Fedora)